src/components/ErrorNotification.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
+
+type Props = {
+  hasError: boolean,
+  // setHasError: (value: boolean) => void,
+  setHasError: React.Dispatch<React.SetStateAction<boolean>>,
+};
+
+export const ErrorNotification: React.FC<Props> = ({
+  hasError, setHasError,
+}) => {
+  const [isPressed, setIsPressed] = useState(false);
+
+  useEffect(() => {
+    setTimeout(() => {
+      setHasError(false);
+    }, 3000);
+  });
```

Comment:
Please, clear the timeout on the component unmount. https://bobbyhadz.com/blog/react-cleartimeout